coalib/bearlib/languages/definitions/VisualBasic.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class VisualBasic:
+    aliases = 'vb',
+    extensions = '.vb', '.bas'
+    comment_delimiter = "'"
```

Comment:
`REM` is another one.
https://docs.microsoft.com/en-us/dotnet/visual-basic/programming-guide/program-structure/comments-in-code

We need to enhance coalib to allow multiple delimiters.  Create an issue please

Reply:
Enhancement done with https://github.com/coala/coala/pull/5595